tests/acceptance/cli/test_deploy.py

```diff
@@ -153,6 +150,25 @@ def test_transfer_tokens(click_runner, registry_filepath):
     # Check that the NU has arrived to the recipient
     assert token_agent.get_balance(address=recipient_address) == 42
 
+    # Let's approve an allowance to a random spender
+    spender_address = get_random_checksum_address()
+    owner_address = testerchain.client.accounts[0]
+    assert token_agent.get_allowance(spender=spender_address, owner=owner_address) == 0
+
+    command = ['transfer-tokens',
+               '--target-address', spender_address,
+               '--value', 42,
+               '--approve',
```

Comment:
Will need to update, if you accept my suggestion about renaming the argument.

Reply:
```suggestion
               '--allowance',
```